src/Template/MailPreview/index.ctp

```diff
@@ -0,0 +1,90 @@
+<?php foreach ($mailPreviews as $plugin => $previews) : ?>
+<h3><?= $plugin ?></h3>
+    <?php foreach ($previews as $preview) : ?>
+        <?php $mailPreview = $preview['class'] ?>
+        <h4>✉️ <?= $mailPreview->name() ?></h4>
```

Comment:
HTML encoding, and what looks like another place where HTML entity version of the emoji would be simpler for other tools[